eng/Versions.props

```diff
@@ -204,8 +204,8 @@
     <MicrosoftWebWebView2Version>1.0.705.50</MicrosoftWebWebView2Version>
     <MicrosoftWebXdtVersion>1.4.0</MicrosoftWebXdtVersion>
     <SystemIdentityModelTokensJwtVersion>6.8.0</SystemIdentityModelTokensJwtVersion>
-    <NuGetVersioningVersion>5.9.0-rc.7121</NuGetVersioningVersion>
-    <NuGetFrameworksVersion>5.9.0-rc.7121</NuGetFrameworksVersion>
+    <NuGetVersioningVersion>5.9.0</NuGetVersioningVersion>
+    <NuGetFrameworksVersion>5.9.0</NuGetFrameworksVersion>
```

Comment:
nit: You're using 5.8.0 for NuGet packages in #31035. Should we be consistent❔

Reply:
Done